packages/backend/src/core/discovery/utils/diffToMessages.ts

```diff
@@ -13,7 +19,9 @@ export function diffToMessages(name: string, diffs: DiscoveryDiff[]): string[] {
 
   const bundledMessages = bundleMessages(messages, maxLength)
 
-  return bundledMessages.map((m) => `${header}${wrapDiffCodeBlock(m)}`)
+  return bundledMessages.map(
+    (m) => `${header}${dependentsMessage}${wrapDiffCodeBlock(m)}`,
```

Comment:
Have you tried how this message will look like in Discord? Please provide a screenshot in PR description

Reply:
Updated